model/convert/service_event.go

```diff
@@ -1014,11 +1014,11 @@ func convertDKGKey(cdcDKGKeys cadence.Value) (crypto.PublicKey, error) {
 	// decode individual public keys
 	pubKeyBytes, err := hex.DecodeString(string(keyHex))
 	if err != nil {
-		return nil, fmt.Errorf("could not decode individual public key into bytes: %w", err)
+		return nil, fmt.Errorf("could not decode hex into bytes: %w", err)
```

Comment:
I would prefer the error text `converting from hex to bytes failed` (using your wording but without mentioning "beacon key"), because the failing function is `DecodeString` which is only converting hex to bytes. 
The error message with context about beacon keys is already provided when the calling function `convertDKGKey` fails.

Reply:
thanks for the explanation and the update of the error message. 👍 very happy with your wording. Will resolve this comment.